DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -67,6 +68,24 @@ public void GetGroupDelegates_returns_expected_delegates()
             }
         }
 
+        [Test]
+        public void GetGroupCourses_returns_expected_courses()
+        {
+            // Given
+            var expectedDateTime = new DateTime(2018, 11, 02, 10, 53, 38, 920);
+            var expectedFirstGroupDelegate = GroupTestHelper.GetDefaultGroupCourse(addedToGroup: expectedDateTime);

```

Comment:
delegate?

Reply:
Just highlighting this in the pool of comments - only var name needs a change though so minor.